kong/pdk/cluster.lua

```diff
@@ -46,7 +46,15 @@ local function new(self)
   --
   -- -- use id here
   function _CLUSTER.get_id()
-    return kong.core_cache:get(CLUSTER_ID_PARAM_KEY, nil, fetch_cluster_id)
+    local cluster_id, err, hit_lvl = kong.core_cache:get(CLUSTER_ID_PARAM_KEY, nil, fetch_cluster_id)
+
+    if err then
+      kong.log.debug("core_cache hit level: ", hit_lvl)
```

Comment:
Can the `hit_lvl` be nil? In which case I don't see value in logging it.

Reply:
It can be `nil` if the `get(...)` is returns `nil, err`.

How about add a `tostring(hit_lvl)`?

Reply:
The log here is not necessary! As when `err` is true, `hit_lvl` is `nil`.

This part is to make sure the `nil` and `hit_lvl` values are are appended to `cluster_id`. Otherwise, `cluster_id` returned is wrong.